docs/eventing/channel-based-broker.md

```diff
@@ -0,0 +1,214 @@
+---
+title: "Channel Based Broker"
+weight: 30
+type: "docs"
+---
+
+# Channel Based Broker
+
+Knative provides a `Broker` implementation that uses [Channels](./channels/) for
+event routing. You will need to have a Channel provider installed, for example
```

Comment:
"Channel CRD" or "Channel implementation" makes more sense to me than "Channel provider" because I'm not sure what provider means here, but I don't have a strong preference.